src/containers/course-section/resources-list/ResourcesList.tsx

```diff
@@ -53,7 +48,35 @@ const ResourcesList: FC<ResourcesListProps> = ({
     sensors
   } = useDndSensor({
     items: itemsForSort,
-    setItems: sortResources,
+    setItems: (newItems) => {
+      sortResources(newItems)
+      if (activeItem) {
+        const activeResource = cooperationData.find(
+          (item) => item.resource.id === activeItem.id
+        )
+        if (
+          activeResource &&
+          activeResource.availability &&
+          updateAvailability
+        ) {
+          updateAvailability(
+            activeResource.resource,
+            activeResource.availability
+          )
+        }
+      }
+      const inactiveResources = cooperationData.filter(
+        (item) => item.resource.id !== activeItem?.id
+      )
+      inactiveResources.forEach((inactiveResource) => {
+        if (inactiveResource.availability && updateAvailability) {
+          updateAvailability(
+            inactiveResource.resource,
+            inactiveResource.availability
+          )
+        }
+      })
+    },
```

Comment:
You should add availability to `CourseResource` type (because each resource conains `availability` field in our db). So if you will change an order of the resource, you will change it with `availability` consistently.

Reply:
Ok, thanks